### DISCUSSION TITLE
Return of slot_asidekick!

### DISCUSSION THREAD

**Discussion by vaniwiki** [combostrap/combo#90] [Ideas]
Would attach itself under the TOC in the right sidebar. Very helpful in those sections where you don't have much TOC content, but the layout still dictates a right sidebar. It would include related content or secondary navigation, tags, and many other things.

Slots are great! Combined with the power of pagequeries, one could populate the asidekick with, well, aside information. It could be used like how Wikipedia has their righthand boxes with an image, tags or keywords, and related stuff at a glance.

Apart from this humble request/idea, the ComboStrap layout and system is perfect.

**Comment by gerardnico** [comment]
Thanks Eidwin :) I understand.

From a design perspective, I tried so much to put all the things in files but I failed.

Because everybody has their own needs.

I'm a great believer in a database where you can get structured data and use them in a [template](https://combostrap.com/theme/what-is-a-template-z9tl87hg#how_to_customize_or_create_a_template).

Mixing styling and structure data in a database (ie file) is a mistake.

Most of the website that runs combo works this way and most slots have only one content (ie page explorer). 
They are just templates, so much work for nothing but yeah.
